portfolio/src/main/webapp/script.js

```diff
@@ -30,3 +30,20 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+var nightModeEnabled = false;
```

Comment:
nit: in looking at the sight currenlty it seems that initally night mode is on by default even when unchecked.  Then after clicking the check, nothing happens (as it is actually still enabled but now it is toggeld on) and then only after this initial check/uncheck does it work as expected